packages/kbn-generate/src/commands/package_command.ts

```diff
@@ -109,14 +114,15 @@ export const PackageCommand: GenerateCommand = {
     } else {
       group = (
         await inquirer.prompt<{
-          group: 'platform' | 'observability' | 'security' | 'search';
+          group: KibanaGroup;
         }>({
           type: 'list',
           choices: [
-            { name: 'Platform', value: 'platform' },
-            { name: 'Observability', value: 'observability' },
-            { name: 'Security', value: 'security' },
-            { name: 'Search', value: 'search' },
+            { name: 'platform', value: 'platform' },
+            ...KIBANA_SOLUTIONS.map((solution) => ({
+              name: solution,
+              value: solution,
+            })),
```

Comment:
nit: should we loop through `KIBANA_GROUPS` and save from having the explicit `platform` entry?